r/vignettes/data_wrangling.Rmd

```diff
@@ -0,0 +1,172 @@
+---
+title: "Data analysis with dplyr syntax"
+description: >
+  Learn how to use the `dplyr` backend supplied by `arrow` 
+output: rmarkdown::html_vignette
+---
+
+The `arrow` package provides a `dplyr` back end that allows users to manipulate tabular Arrow data (`Table` and `Dataset` objects) using familiar `dplyr` syntax. To use this functionality, make sure that the `arrow` and `dplyr` packages are both loaded. In this article we will take the `starwars` data set included in `dplyr`, convert it to an Arrow Table, and then analyze this data. Note that, although these examples all use an in-memory `Table` object, the same functionality works for an on-disk `Dataset` object with only minor differences in behavior (documented later in the article).
```

Comment:
Could we use a different term to "back end" here? I've heard different people use the terms "back end", "frontend", "API" and other terms, and I think this can sound a bit ambiguous.

Reply:
Hm yeah. Okay, what I've done is refer only to "functionality". It's still ambiguous, but nobody expects that term to be precise. I figure that's better than trying to describe what the arrow/dplyr bindings actually are in the opening sentence!